signing-manifests/test-mac-hardened-sign.yml

```diff
@@ -0,0 +1,34 @@
+---
+bug: 0000000
+sha256: b168118ad05f0fac9b48232c7b2b3d2a723761f8e8c20e722496ff941e352b87
+filesize: 148397294
+private-artifact: false
+signing-formats: ["macapp", "autograph_widevine", "autograph_omnija"]
+requestor: Heitor Neiva <hneiva@mozilla.com>
+reason: Firefox hardened signing
+product: firefox
+artifact-name: target.dmg
+mac-behavior: mac_sign_hardened
+signingscript-notarization: false
+hardened-sign-config:
+  - deep: false
+    runtime: true
+    force: true
+    entitlements: https://hg.mozilla.org/mozilla-central/raw-file/tip/security/mac/hardenedruntime/browser.production.entitlements.xml
+    globs:
+      - "/Contents/MacOS/crashreporter.app"
```

Comment:
Some of these are folders. I'll confirm with Haik what the behavior should be for these (likely the globs should be adjusted)

Reply:
There was actually a bug in the check function.
`pathlib.Path` doesn't like joining things like `/foo/bar` and `/hello/*` because both start with `/`
Should be good now.

FWIT no business logic has changed by this. Just the actual glob check functions.